doc/_admin-guide/070_Destinations/300_syslog/README.md

```diff
@@ -70,6 +70,6 @@ destination d_syslog_tls {
 ```
 
 **NOTE:** If a message uses the IETF-syslog format (RFC-5424), only the text
-of the message can be customized (that is, the ${MESSAGE} part of the
+of the message can be customized (that is, the MESSAGEs part of the
```

Comment:
is this intentionally plural? (only one log in the sentence, and a log only has one message)

Reply:
Sorry, this was unintentional, fixed now.